src/editor/editor.cpp

```diff
@@ -54,15 +68,18 @@ Uint32 imgui_save_notification_timer = 0;
 Uint32 imgui_tileset_n_rows = 8;
 Uint32 imgui_tileset_n_cols = 20;
 Uint32 tile_cell_size;
+Uint32 mouse_button_state_current;
+Uint32 mouse_button_state_prev;
 char *imgui_tilemap_save_notification_text;
 char *imgui_tilemap_save_notification_text_success = (char *) "Saved Successfully!";
 char *imgui_tilemap_save_notification_text_failure = (char *) "Save Failed!!!!";
 
-std::filesystem::path assets_rel_path_prefix = "assets/";
+std::filesystem::path assets_rel_path_prefix = "../assets/";
```

Comment:
get rid of the `../`

Reply:
you'll need to change the working directory of when you're running the executable